envoy/network/filter.h

```diff
@@ -400,6 +404,85 @@ class ListenerFilterManager {
  */
 using ListenerFilterFactoryCb = std::function<void(ListenerFilterManager& filter_manager)>;
 
+/**
+ * QUIC Listener Filter
+ */
+class QuicListenerFilter {
```

Comment:
Can this class be a child class of above class ListenerFilter ?

Reply:
The contract is quite different from ListenerFilter though.

Reply:
ACK